CHANGELOG.rst

```diff
@@ -34,6 +34,12 @@ Removed
 this functionality. This required updating a lot of test files.
 - The `columns_set_or_check()` method from BaseTransformer. With the above change it was no longer necessary. Subsequent updates to nominal transformers and their tests were required.
 
+1.2.3 (2024-02-23)
```

Comment:
Changelog is out of date, please can you pull the latest changes from develop to your branch :)

Reply:
main is on 1.3.0 now: 
 https://github.com/lvgig/tubular/blob/main/CHANGELOG.rst

Reply:
Can you move this entry to under 1.3.0 please @ChaitanMohr ?

Reply:
https://github.com/lvgig/tubular/pull/183/commits/d1d8044bf28cb5a0cc02eef9ce2f50ecad719cdf

Reply:
Hm looks like the changelog is unchanged?

Reply:
updated changelog https://github.com/lvgig/tubular/pull/183/commits/2544a1f132029d839c7c2374c2a40e93c86c6da5